### ISSUE TITLE
missing ./.cache/spl 2-battle.json files

### ISSUE THREAD

**Issue by AntiPingMaster** [sflavelle/neubott#15]
I have a question, where should we get these files, see picture, from?
![image](https://user-images.githubusercontent.com/101053761/157226285-dcf75630-37b9-497c-9770-2ad8f89c4bb7.png)
https://i.imgur.com/vyFiGTX.png

**Comment by sflavelle** [comment]
Hi,

TL;DR you shouldn't need the cache files, if axios is installed (which it should be when installing this bot's dependencies) it will automatically fetch the stage data and cache it.

Assuming you are just using the command from this repository and not the bot itself, you will need to install [axios](https://www.npmjs.com/package/axios) as a dependency. This HTTP client will then fetch the appropriate data and then cache the file to a `.cache` folder in your bot's root.

---

https://github.com/sflavelle/neubott/blob/c8e20aa01b68944da596923b5c1db70b5701a81b/commands/splatoon.js#L41

The main files for pulling data from Splatoon 2 come from the splatoon2.ink repository: https://github.com/misenhower/splatoon2.ink/wiki/Data-access-policy

If the cache doesn't exist or is out of date, the data will be pulled from splatoon2.ink and then be saved locally as a cache file. This is so that if many people are using the command during the two hour period between stage rotations, we are not constantly hitting the Splatoon2.ink server.
